### PR TITLE
Initial layout

### PR DESCRIPTION
**Pull request by iCherya**
Dear, please check

### CODE REVIEW

**Comment by iCherya** [comment]
Guys,
I've fixed some layout issues that we discussed yesterday.
Please, take a look

File: src/styles/_mixins.scss [hunk 1]
```diff
@@ -0,0 +1,78 @@
+@mixin link {
+  color: $company-green;
+  font-size: inherit;
+  font-family: $main-font;
+  background-color: transparent;
+  border: none;
+  outline: none;
+  text-decoration: underline;
```
Comment (itbasket) [inline src/styles/_mixins.scss]: text-decoration property defined two times and will be overwritten to 'none'

**Comment by itbasket** [comment]
Good job in general, please also add some favicon to the site.

File: src/styles/_mixins.scss (hunk 1 above)
Comment (iCherya) [reply src/styles/_mixins.scss]: Fixed

**Comment by iCherya** [comment]
Guys, fixed issues

**Review by itbasket — APPROVED** [review]
Good job

**Pull request merged**